README.md

```diff
@@ -35,7 +35,7 @@ Then:
 
 📦 [Install the usercss](https://github.com/StylishThemes/StackOverflow-Dark/raw/master/stackoverflow-dark.user.css). Supports automatic updates.
 
-## Additional styles
+## Additional styles (required for properly rendering blocks of code on StackOverflow pages)
```

Comment:
```suggestion
## Additional styles
```

Not all additional styles will be required. This one is thats it.

Reply:
Yes, that is correct. Didn't look at it that way. :+1: